From a6fc42c86864175830e6f211b4d0cb6ea4b7b287 Mon Sep 17 00:00:00 2001
From: Yang Shi <yang.shi@windriver.com>
Date: Mon, 9 Sep 2013 15:35:29 -0700
Subject: [PATCH] trace-cmd: Remove prefix before $(libdir) to install plugins
 into right dir

The recipe defines prefix to "/usr", $(libdir) is "/usr/lib" in bitbake build
system, so the plugins will be installed to "/usr/usr/lib" instead of "/usr/lib".

So, remove $(prefix) before $(libdir) in Makefile to correct install path for
plugins.

Signed-off-by: Yang Shi <yang.shi@windriver.com>
---
 Makefile |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 1964949..3fe2fa8 100644
--- a/Makefile
+++ b/Makefile
@@ -57,8 +57,8 @@ ifeq ($(prefix),$(HOME))
 plugin_dir = $(HOME)/.trace-cmd/plugins
 python_dir = $(HOME)/.trace-cmd/python
 else
-plugin_dir = $(prefix)/$(libdir)/trace-cmd/plugins
-python_dir = $(prefix)/$(libdir)/trace-cmd/python
+plugin_dir = $(libdir)/trace-cmd/plugins
+python_dir = $(libdir)/trace-cmd/python
 PLUGIN_DIR = -DPLUGIN_DIR="$(plugin_dir)"
 PYTHON_DIR = -DPYTHON_DIR="$(python_dir)"
 PLUGIN_DIR_SQ = '$(subst ','\'',$(PLUGIN_DIR))'
-- 
1.7.5.4

